models/public/efficientdet-d1-tf/efficientdet-d1-tf.md

```diff
@@ -55,22 +55,21 @@ The array of summary detection information, name: `detections`, shape: [1, N, 7]
 bounding boxes. For each detection, the description has the format:
 [`image_id`, `y_min`, `x_min`, `y_max`, `x_max`, `confidence`, `label`],
     where:
-
+   where:
     - `image_id` - ID of the image in the batch
     - (`x_min`, `y_min`) - coordinates of the top left bounding box corner
     - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner
     - `confidence` - confidence for the predicted class
-    - `label` - predicted class ID, starting from 1
+    - `label` - predicted class ID, in range [1, 91] across following [labels](../../../data/dataset_classes/coco_91cl.txt)
 
 ### Converted Model
 
 The array of summary detection information, name: `detections`, shape: [1, 1, N, 7], where N is the number of detected
 bounding boxes. For each detection, the description has the format:
 [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`],
     where:
-
     - `image_id` - ID of the image in the batch
-    - `label` - predicted class ID, starting from 0
+    - `label` - predicted class ID, in range [0, 90] across following [labels](../../../data/dataset_classes/coco_91cl.txt)
```

Comment:
```suggestion
    - `label` - predicted class ID, in range [0, 90], mapping to class names provided in  `<omz_dir>/data/dataset_classes/coco_91cl.txt` file
```

Reply:
I believe link on file is better than just path to it, but if you want, ok I can replace

Reply:
relative path will not work in openvino online documentation. Roman will do something with this but not in this release